el_hub/el_hub_flutter/lib/src/widgets/power_installations/power_installation_graph_stream_builder.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:el_hub_flutter/src/widgets/power_installations/power_installation_graph.dart';
+import 'package:el_hub_flutter/src/widgets/power_installations/power_installation_graph_section.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:flutter/src/widgets/placeholder.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../../bloc/power_installations_bloc.dart';
+import '../../../utility/classes/chart_sample_data.dart';
+
+class PowerInstallationGraphStreamBuilder extends StatelessWidget {
+  const PowerInstallationGraphStreamBuilder({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: SizedBox(
+        child: StreamBuilder<PowerInstallationsState>(
+          stream: BlocProvider.of<PowerInstallationsBloc>(context)
+              .powerInstallationStateStream,
+          builder: (context, snapshot) {
+            if (snapshot.connectionState == ConnectionState.waiting) {
+              return const CircularProgressIndicator();
+            } else if (snapshot.connectionState == ConnectionState.done) {
+              return const Text('done');
+            } else if (snapshot.hasError) {
+              return const Text('Error!');
+            } else if (snapshot.data == null) {
+              return const Text('No Data');
+            } else {
+              return ListView.builder(
+                itemCount: snapshot.data?.powerInstallations.length ?? 0,
+                itemBuilder: (context, index) {
+                  return PowerInstallationGraphSection(
+                    powerInstallation: snapshot.data!.powerInstallations[index],
+                  );
+                },
+              );
```

Comment:
God brugervenlighed hvis nu at conntionstate har fejl :)

KEEP UP THE UMBAZING WORK!!